packages/playground/package.json

```diff
@@ -54,12 +54,12 @@
     "less": "^3.11.3",
     "less-loader": "^5.0.0",
     "lodash": "^4.17.21",
+    "nanoid": "^3.1.23",
```

Comment:
We should be able to remove this dependency (because nanoid is already a dependency pulled in by @rjsf/core, etc.). So we can revert package.json and package-lock.json just for the playground directory.

Reply:
I reverted the package-lock changes and removed the shortid dependency